manifests/init.pp

```diff
@@ -165,6 +170,10 @@
   $kadmind_enable = true,
   $kadmind_acls = { "*/admin@$realm" => '*' },
 
+  $host_ticket_cache_ccname = "/tmp/krb5cc.puppet",
```

Comment:
Without PKINIT kinit will need a keytab which would need to be created using kadmin first, atm manually. Automating kadmin unfortunately also requires a keytab. Of course, passing passwords to kinit and/or kadmin could also be implemented. But I intentionally didn't do that because it's inherently insecure. I'm still trying to get rid of it for KDC database creation.

Re temp file names: MIT Kerberos and Heimdal have been putting their ticket caches with predictable names like krb5cc_<uid> into /tmp for decades now. I just assumed they had secured their tools accordingly. Since I'm not the guy to do a code review on this and this is our very own special host ticket cache, I guess we could just put it somewhere else.

How about /var/lib/puppet/krb5cc.puppet? The directory should already exist and the file name is specific enough to be unlikely to collide with anything puppet puts there. /var/cache/krb5cc.puppet also seems a nice choice. You decide. :)